LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Nodes Agency - Android
```

Comment:
Should this say Monstar Lab? 🧐

Reply:
https://github.com/nstack-io/php-sdk/blob/master/LICENSE 😄